.github/workflows/maven.yml

```
@@ -36,7 +36,7 @@ jobs:
       uses: actions/setup-java@v4
       with:
         java-version: 17
-        distribution: adopt
+        distribution: adopt-openj9
```

```suggestion
        distribution: temurin
```

Maybe try `temurin`, as in [Struts](https://github.com/apache/struts/blob/master/.github/workflows/maven.yml#L42)?

---

Yes, I already rebased and changed this